workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/Injection/RequiredFieldInjector.cs

```diff
@@ -41,7 +43,7 @@ public RequiredFieldInjector(EntityManager entityManager, ILogDispatcher logger)
         public bool HasRequiredFields(Type behaviourType)
```

Comment:
Maybe rename this method to something like `IsSpatialOSBehaviour` in the sense that the Behaviour should be managed by us.